src/async-components/structures/CompatibilityView.tsx

```diff
@@ -0,0 +1,408 @@
+// scss variables will be accessible.
+import "../../../res/css/structures/ErrorView.scss";
+
+interface IProps {
+    onAccept: () => void;
+}
+
+const CompatibilityView: React.FC<IProps> = ({ onAccept }) => {
+    return <div className="mx_ErrorView">
+        <div className="mx_ErrorView_container">
+            <div className="mx_HomePage_header">
+                <span className="mx_HomePage_logo">
+                    <img height="42" src="themes/riot/img/logos/riot-logo.svg" alt="Riot" />
+                </span>
+                <h1>{ _t("Unsupported browser") }</h1>
+            </div>
+
+            <div className="mx_HomePage_col">
+                <div className="mx_HomePage_row">
+                    <div>
+                        <h2 id="step1_heading">{ _t("Your browser can't run Riot") }</h2>
+                        <p>
+                            { _t(
+                                "Riot uses advanced browser features which aren't supported by your current browser.",
+                            ) }
+                        </p>
+                        <p>
+                            { _t(
+                                'Please install <chromeLink>Chrome</chromeLink>, <firefoxLink>Firefox</firefoxLink>, ' +
+                                'or <safariLink>Safari</safariLink> for the best experience.',
+                                {},
+                                {
+                                    'chromeLink': (sub) => <a href="https://www.google.com/chrome">{sub}</a>,
+                                    'firefoxLink': (sub) => <a href="https://firefox.com">{sub}</a>,
+                                    'safariLink': (sub) => <a href="https://apple.com/safari">{sub}</a>,
+                                },
+                            )}
+                        </p>
+                        <p>
+                            { _t(
+                                "You can continue using your current browser, but some or all features may not work " +
+                                "and the look and feel of the application may be incorrect.",
+                            ) }
+                        </p>
+                        <button onClick={onAccept}>
+                            { _t("I understand the risks and wish to continue") }
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div className="mx_HomePage_col">
+                <div className="mx_HomePage_row">
+                    <div>
+                        <h2 id="step2_heading">Use Riot on mobile</h2>
+                        <p><strong>iOS</strong> (iPhone or iPad)</p>
+                        <a href="https://itunes.apple.com/app/riot-im/id1083446067?mt=8" target="_blank"
+                           className="mx_ClearDecoration">
+                            <svg width="144px" height="48px" viewBox="0 0 120 40" version="1.1"
```

Comment:
Why do we have an entire SVG here when we should be importing it?

Applies throughout.